### ISSUE TITLE
Fix Figure 1c, #assemblages

### ISSUE THREAD

**Issue by ahhurlbert** [hurlbertlab/core-transient#103]
The fact that the number of bird assemblages is >10000 makes me think that this figure is portraying the number of assemblages including each different spatial scale as another assemblage.

I think it would be more useful to convey the number of assemblages at the one scale chosen for analysis in the non-scale portions of the paper.

**Comment by ssnell6** [comment]
it looks like only the bbs included all the scales, not sure why we chose to do it that way before. I subsetted to the route scale level bbs dataset, new figure is in output.

**Comment by ahhurlbert** [comment]
@ssnell6 I don't think this has been addressed yet. Still need to adjust Fig 2 accordingly, as it must still be counting different scales of the same assemblage.

**Comment by ssnell6** [comment]
The label in 2b is wrong - the terrestrial # of assemblages is 1462. I fixed the label in the repo and MS
